dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/CompanyApi.kt

```diff
@@ -260,4 +260,46 @@ interface CompanyApi {
         produces = ["application/json"],
     )
     fun getTeaserCompanies(): List<String>
+
+    /**
+     * A method used to retrieve the aggregated meta information for all frameworks
+     * @param companyId the identifier of the company to collect the information for
+     * @returns the collected aggregated meta information per framework
+     */
+    @Operation(
+        summary = "Retrieve aggregated meta information for all frameworks",
+        description = "For each framework retrieves the amount of availabe reporting periods",
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Successfully retrieved values."),
+        ],
+    )
+    @GetMapping(
+        value = ["/{companyId}/aggregated-framework-meta-info"],
+        produces = ["application/json"],
+    )
```

Comment:
what does this exactly do?

Reply:
The endpoint will give you an overview for the specified company. This overview contains the number of connected datasets sorted by the frameworks. For example it tells you something like sme=0, lksg=5, p2p=1, etc